src/app/space/create/deployments/components/deployment-card/deployment-card.component.html

```diff
@@ -1,11 +1,11 @@
 <div class="card-pf" (click)="collapsed = !collapsed">
   <div class="card-pf-body">
     <span class="pficon pficon-ok" tooltip="Everything looks okay" placement="top"></span>
-    <span id="podCountLabel">{{ podCount | async }} Pods</span>
+    <deployments-donut [applicationId]="applicationId" [mini]="true" ></deployments-donut>
     <span id="versionLabel">{{ version | async }}</span>
     <div class="pull-right dropdown-kebab-pf dropdown" dropdown (click)="$event.stopPropagation()">
-      <button class="btn btn-link pull-right dropdown-toggle" id="kebab" type="button" aria-haspopup="true"
-        aria-expanded="true" data-toggle="dropdown" dropdownToggle>
+      <button class="btn btn-link pull-right dropdown-toggle" id="kebab" type="button" aria-haspopup="true" aria-expanded="true"
+        data-toggle="dropdown" dropdownToggle>
```

Comment:
Is this change fixing a line length style linting issue or something?

Reply:
Oh, my bad. I suppose it's my IDE's auto formatting. I don't think there's linting on html files; I'll revert this change.